README.md

```diff
@@ -109,7 +109,9 @@ backwards-compatibity.
 
 ## Poller
 Telebot doesn't really care how you provide it with incoming updates, as long
-as you set it up with a Poller:
+as you set it up with a Poller, or call ProcessUpdate for each update (see
```

Comment:
The provided example is somehow removed later on. What happened? Is it no longer working? thx!

Reply:
> `tb "gopkg.in/tucnak/telebot.v2"` 

Oh, it's for V2 only, and not working in V3 any more?